.gitignore

```diff
@@ -7,4 +7,11 @@ npm-debug.log
 *.vsix
 !sbt/src/server-test/completions/target
 .big
+
+# eclipse, intellij, bloop, metals
+.classpath
+.project
 .idea
+.settings
+.bloop
+.metals
```

Comment:
Tooling specific stuff should be in a global gitignore of each user. I've only conceded to `.idea` because that's what most new users seem to use.
https://help.github.com/en/articles/ignoring-files

Reply:
ahhh gotcha! 😄 I'll close this.